.github/dependabot.yml

```
@@ -8,7 +8,8 @@ updates:
   - package-ecosystem: "npm" # See documentation for possible values
     directory: "/" # Location of package manifests
     schedule:
-      interval: "daily"
+      interval: "weekly"
+      day: "wednesday"
```

According to the documentation, it looks like the job runs on a random time when the time is not specified. 

Would it be better if we specify the time so that the dependabot PRs are ready at the start of the day? Something like below? (Trying to set that the job runs at the 00:00 AM on Wednesday) 

```suggestion
      interval: "weekly"
      day: "wednesday"
      time: "00:00"
      timezone: "Europe/Berlin"
```